website/docs/guides/orchestration/running-dbt-cloud-jobs/3-anti-patterns.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Anti Patterns for dbt Cloud jobs"
+id: "dbt-cloud-job-anti-patterns"
+slug: 3-anti-patterns
+description: Recognize common anti-patterns in dbt Cloud jobs
+displayText: dbt Cloud job anti-patterns
+hoverSnippet: Recognize common anti-patterns in dbt Cloud jobs.
+---
+
+dbt's ref() function enables running data models in the proper sequence automatically and in parallel. Thus, it's possible to make things harder than they need to be by making dbt Cloud job runs too specific. See organizing-dbt-cloud-jobs for our recommendations on approach.
+
+### Example DAG
+
+Model `fct_orders` ref()s `int_orders` which ref()s `stg_orders`
```

Comment:
```suggestion
Model `fct_orders` refs `int_orders` which refs `stg_orders`
```

Or even better, a screenshot of a DAG that actually shows this in action. Picture, 1000 words, etc